### PR TITLE
Add contact form and footer to the page

### PR DESCRIPTION
**Pull request by EidHachem**
Add contact section
Add footer with links to social media accounts
Make page mobile friendly

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -220,5 +231,45 @@ <h3 class="project-title">Multi-Post Stories</h3>
...
+        <input type="text" name="Name" id="name" maxlength="30" placeholder="Text Input" />
+        <input type="email" name="Email" id="mail" placeholder="Jeremy@gmail.com" />
+        <textarea
+          name="user_message"
+          id="message"
+          maxlength="500"
+          placeholder="Enter text here"
+        ></textarea>
+        <button class="form-button" type="submit">See Project</button>
+      </form>
```
Comment (sja-thedude) [inline index.html]: - [x] Kindly fix the form as it is not submitting.
Screenshot 2022-03-22 at 11 57 58 PM

File: style.css [hunk 2]
```diff
@@ -301,11 +299,10 @@ hr {
...
 }
 
-@media only screen and (max-width: 400px) {
+@media only screen and (max-width: 375px) {
   body {
-    width: 100%;
+    width: 375px;
     margin: 1px 2px;
-    overflow: scroll;
   }
```
Comment (sja-thedude) [inline style.css]: - Kindly remove media for the mobile version. This is used in the desktop version and used as min-width. This gives problems to your code.

File: style.css [hunk 3]
```diff
@@ -497,4 +494,79 @@ hr {
...
+  }
+
+  .footer-email {
+    font-family: roboto, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    color: #ebebff;
+    margin-top: 0;
+  }
 }
```
Comment (sja-thedude) [inline style.css]: - [x] Kindly fix the layout (colours, font, size, space, margins) according to the Figma design. You can open the browser as chrome from vscode and fix it.
- [x] Kindly add button interactions such as hover, active when pressed according to the Figma design.
- Your design:
Screenshot 2022-03-22 at 11 58 38 PM

- Figma design:
Screenshot 2022-03-23 at 12 05 47 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @EidHachem, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good commit message
✔️ Good pull request title

## Required 
- [x] Kindly add collaborators' info in your readme file. 
- [x] Make sure to add a descriptive PR description.
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**